docs/cookbook/payments/customizing-credit-memo.rst

```diff
@@ -21,6 +21,17 @@ The first exemplary customization is to change background color of the heading o
             /* ... */
         </style>
 
+Customizing Credit Memo's logo
+----------------------------------
+
+Credit Memo's logo is by default displayed as Sylius logo.
+Changing it is done by modifying the ``SYLIUS_REFUND_LOGO_FILE`` environment variable.
+You can achieve that by overwriting it's path in your ``.env`` file, like in the example below:
+
+    .. code-block:: text
+
+        SYLIUS_REFUND_LOGO_FILE=/Users/Sylius/logo_directory/your_logo.png
```

Comment:
Similar to https://github.com/Sylius/Sylius/pull/13421#pullrequestreview-843431184, you should put a custom logo in the project directory and then use it with the proper path.